internal/pkg/deploy/cloudformation/stack/testdata/environments/template-with-defaultvpc-flowlogs.yml

```
@@ -1003,14 +1003,20 @@ Resources:
       DomainName: !Ref AppDNSName
       PublicAccessDNS: !GetAtt PublicLoadBalancer.DNSName
       PublicAccessHostedZone: !GetAtt PublicLoadBalancer.CanonicalHostedZoneID
+  LogGroup:
```

Can we rename the logical ID to be more specific to flow logs:
```suggestion
  VpcFlowLogGroup:
```